CHANGELOG.md

```diff
@@ -10,22 +10,30 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 - Block Teleporter
 - Built in compatability for Better Combat mod
-
+- Cindered ladder and bookshelf
+- Shadewood ladder and bookshelf
+- Parrots will mimic imps, ogres, basilisks, and liches
+- 
```

Comment:
```suggestion

```
There's also an extra empty line added under "Fixed" that would be nice to clean up.